mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -378,7 +378,7 @@ CLASS net/minecraft/class_864 net/minecraft/entity/Entity
 	METHOD method_2515 isSwimming ()Z
 	METHOD method_2516 getAir ()I
 	METHOD method_2517 setInLava ()V
-	METHOD method_2518 getTranslationKey ()Ljava/lang/String;
+	METHOD method_2518 getTranslatedName ()Ljava/lang/String;
```

Comment:
hmmm, it returns the translation key or the translated name depending on the version, what should we do?

Reply:
Call it getName or simply go back to having multiple branches :tiny_potato:

Reply:
I think getName could work if it's not taken.